### PR TITLE
SDCICD-190. Handle OCM API metrics, test files.

### PR DESCRIPTION
**Pull request by meowfaceman**

### CODE REVIEW

File: scripts/metrics-sync.sh [hunk 1]
```diff
@@ -13,7 +13,7 @@ PUSHGATEWAY_URL=${PUSHGATEWAY_URL%/}
 trap 'rm -rf "$VENV" "$METRICS_DIR"' EXIT
 
 # First, we should detect any stale metrics and purge them if needed
-METRICS_LAST_UPDATED=$(curl "$PUSHGATEWAY_URL/metrics" | grep "^push_time_seconds{.*" | grep osde2e | sed 's/^.*job="\([[:alnum:]_.-]*\)".*\}\s*\(.*\)$/\1,\2/' | sort | uniq)
+METRICS_LAST_UPDATED=$(curl "$PUSHGATEWAY_URL/metrics" | grep "^push_time_seconds{.*" | grep -E 'osde2e|ocm-api-test' | sed 's/^.*job="\([[:alnum:]_.-]*\)".*\}\s*\(.*\)$/\1,\2/' | sort | uniq)
```
Comment (markturansky) [inline scripts/metrics-sync.sh]: yes, `ocm-api-test` is what I am using

File: scripts/metrics-sync.sh [hunk 2]
```diff
@@ -52,19 +52,28 @@ for file in $METRICS_FILES; do
 	fi
 
 	JOB_NAME=$(echo $file | sed 's/^[^\.]*\.\(.*\)\.metrics\.prom$/\1/')
-	if ! curl -X DELETE "$PUSHGATEWAY_URL/metrics/job/$JOB_NAME"; then
-		echo "Error deleting old results for $JOB_NAME."
-		exit 3
-	fi
+	if [[ ! $JOB_NAME = delete_* ]]; then
```
Comment (markturansky) [inline scripts/metrics-sync.sh]: yes, `delete_` is my prefix for testing uploads to S3

**Comment by markturansky** [comment]
confirmed my two strings.  LGTM.

**Comment by jeefy** [comment]
Merged manually. Prow verify ran successfully.

**Pull request merged**